hazelcast/src/main/java/com/hazelcast/monitor/impl/LocalMapStatsImpl.java

```diff
@@ -390,6 +390,7 @@ public String toString() {
                 + ", lockedEntryCount=" + lockedEntryCount
                 + ", dirtyEntryCount=" + dirtyEntryCount
                 + ", heapCost=" + heapCost
+                + ", nearCacheStats=" + nearCacheStats
```

Comment:
since `nearCacheStats` can be `null` when there is no near cache enabled, 
doing `nearCacheStats == null ? "" :  ", nearCacheStats=" + nearCacheStats` can be more appropriate